docs/getting-started/defend-feature-privs.asciidoc

```diff
@@ -47,6 +47,11 @@ To grant access, select *All* for the *Security* feature in the *{kib} privilege
 
 | *File Operations*
 | Perform file-related <<response-actions,response actions>> in the response console.
+
+| *Execute Operations*
+a| Perform shell commands and script-related <<response-actions,response actions>> in the response console.
+
+WARNING: The commands are run on the host using the same user account running the {elastic-defend} integration, which normally has full control over the system. Restrict this feature privilege to only {elastic-sec} users who require this level of access.
```

Comment:
Made some slight modifications to the second sentence to hopefully make it more straightforward. 

```suggestion
WARNING: The commands are run on the host using the same user account running the {elastic-defend} integration, which normally has full control over the system. Only grant this feature privilege to {elastic-sec} users who require this level of access.
```